libpq/plan.sh

```diff
@@ -0,0 +1,74 @@
+pkg_name=libpq
+pkg_origin=core
+pkg_version=9.6.6
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="LibPQ is the client side library for PostgreSQL, a powerful, open source object-relational database system."
+pkg_upstream_url="https://www.postgresql.org/"
+pkg_license=('PostgreSQL')
+pkg_source=https://ftp.postgresql.org/pub/source/v${pkg_version}/postgresql-${pkg_version}.tar.bz2
+pkg_dirname=postgresql-${pkg_version}
+pkg_filenam=postgresql-${pkg_version}.tar.gz
+pkg_shasum=399cdffcb872f785ba67e25d275463d74521566318cfef8fe219050d063c8154
+
+pkg_deps=(
+  core/bash
```

Comment:
If the intent is for client-side applications to use libpq as a library, does it need to explicitly include bash? Or does the lib link against bash?